tools-local/tasks/mobile.tasks/CreateWasmBundle/CreateWasmBundle.csproj

```diff
@@ -10,7 +10,9 @@
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(RefOnlyMicrosoftBuildTasksCoreVersion)" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(RefOnlyMicrosoftBuildUtilitiesCoreVersion)" />
   </ItemGroup>
-
+  <ItemGroup>
+    <PackageReference Include="System.Runtime.TimeZoneData" Version="$(SystemRuntimeTimeZoneDataVersion)" GeneratePathProperty="true" PrivateAssets="all" />
```

Comment:
Last thing - you can drop `GeneratePathProperty=..` now, since you are not using the generated path property `$(PkgSystem_Runtime_TimeZoneData)` now.